doc/bootstrap.rst

```diff
@@ -144,6 +144,27 @@ deploy one or more new manager daemons,::
 Deploying MDSs
 ==============
 
-In order to use the CephFS file system, one or more MDS daemons is needed.
+In order to use the CephFS file system, one or more MDS daemons is
+needed.  Normally these are created automatically if the newer ``ceph
+fs volume`` interface is used to create a new file system.  For more information, see :ref:`fs-volumes-and-subvolumes`.
 
-TBD
+To explicitly deploy metadata servers,::
```

Comment:
```suggestion
To deploy metadata servers, execute the following::

Reply:
Note: try to avoid superfluous language such as "explicity". Whilst it is a specific term and tells the user to _explicitly_ deploy the metadata servers, saying "To deploy metadata servers" provides the exact same instruction. That, or "To only deploy..."